### PR TITLE
Do not attempt to restore db if evmserverd is running

### PR DESCRIPTION
**Pull request by yrudman**
Do not attempt to restore db if evmserverd is running

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1662991

### CODE REVIEW

**Comment by yrudman** [comment]
@miq-bot add-label enhancement, changelog/yes

File: lib/manageiq/appliance_console/database_admin.rb [hunk 1]
```diff
@@ -43,6 +43,11 @@ def object_store_backup?
 
       def ask_questions
         setting_header
+        if action == :restore && LinuxAdmin::Service.new("evmserverd").running?
+          say("\nDatabase #{action} failed. Please execute \“Stop EVM Server Processes\” command and try again.")
```
Comment (carbonin) [inline lib/manageiq/appliance_console/database_admin.rb]: Can't action only be restore here?

Comment (carbonin) [reply lib/manageiq/appliance_console/database_admin.rb]: Also I think better grammar would be `Please execute the "Stop EVM Server Processes" command and try again`

Comment (yrudman) [reply lib/manageiq/appliance_console/database_admin.rb]: ```actions```: could be ```:dump``` here

Comment (carbonin) [reply lib/manageiq/appliance_console/database_admin.rb]: Not according to the conditional ...

Comment (yrudman) [reply lib/manageiq/appliance_console/database_admin.rb]: @carbonin by here I meant inside ```#ask_questions```,
 Is your question about changing ```#{action}``` to "restore"

Comment (carbonin) [reply lib/manageiq/appliance_console/database_admin.rb]: > Is your question about changing #{action} to "restore"

Yes

**Comment by carbonin** [comment]
@yrudman can you remove the "Fixes" from the commit message because this won't actually resolve the BZ until a version of the console is released and included into a new build?

**Comment by yrudman** [comment]
@carbonin ok, commit message changed

**Pull request merged**